openhtf/exe/phase_executor.py

```diff
@@ -32,21 +32,23 @@
 framework.
 """
 
+import argparse
 import collections
 import inspect
 import logging
 
-import gflags
-
 import openhtf
 from openhtf.exe import phase_data
 from openhtf.io import test_record
+from openhtf.util import argv
 from openhtf.util import threads
 
-
-FLAGS = gflags.FLAGS
-gflags.DEFINE_integer('phase_default_timeout_ms', 3 * 60 * 1000,
-                      'Test phase timeout in ms', lower_bound=0)
+ARG_PARSER = argv.ModuleParser()
+ARG_PARSER.add_argument(
+    '--phase_default_timeout_s', default=3 * 60, action=argv.StoreInModule,
```

Comment:
I didn't even notice that `default=` in the code, the github coloring is better than my vim setup :/